### PR TITLE
Added a generic error page, displaying error code

### LINKED ISSUE #75: As a User, I want to see a custom 404 page when I'm lost.
### As a user, I want to see a custom 404 page when I'm lost on the website. 

- It will require a lot of front-end work. You can reuse some template in twig already existing to do this. 
- It will require some back-end work to enable the fact that we can use a custom 404 page. 

Here's the design for the 404 page: 
73266562-64045100-41d7-11ea-9317-20bf3810485c

### PR DESCRIPTION
**Pull request by me-v-en**
resolve #75 - Added a generic error page, displaying error code (404, 500, etc.)
 * If a specific page is needed for an error, add a new twig file : templates/bundles/TwigBundle/Exception/error<error-code>.html.twig

### CODE REVIEW

**Comment by Noelierx** [comment]
Hello @me-v-en Thank you a lot for your pull request, we just need to verify it, but don't worry even if this PR isn't merged at the end of your counter, it will still be a part for the Hacktoberfest and your personal counter!

File: templates/bundles/TwigBundle/Exception/error.html.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		<div class="container">
+				<div class="col error__texts">
+					<h1 class="error__text">
+						<span class="error__text__title">{{ 'error.title' |trans }}</span>
+						<span class="error__text__code">{{ status_code }}</span>
+					</h1>
+						<a class="btn roundBorder error__button " href="{{ path('index') }}" rel="home">{{ 'error.button.text' |trans }}</a>
+
+				</div>
+				<div class="error__illustration">
```
Comment (Noelierx) [inline templates/bundles/TwigBundle/Exception/error.html.twig:17]: I don't see any CSS in the "errors.css" file about this class, is this normal?

File: assets/css/errors.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	margin: 0;
+	margin-top: 5rem;
+}
+
+.error__text__title {
+	display: inline-block;
+	writing-mode: vertical-rl;
+	transform-origin: center;
+	transform: rotate(180deg);
+
```
Comment (Noelierx) [inline assets/css/errors.css:84]: Unnecessary blank space

File: assets/css/errors.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	text-transform: uppercase;
+	position: relative;
+
```
Comment (Noelierx) [inline assets/css/errors.css:87]: same

**Review by Noelierx — CHANGES_REQUESTED** [review]

File: templates/bundles/TwigBundle/Exception/error.html.twig (hunk 1 above)
Comment (me-v-en) [reply templates/bundles/TwigBundle/Exception/error.html.twig]: There isn't, I wrote it expecting to style the element, but it wasn't necessary. 
I removed the class.

**Review by Noelierx — APPROVED** [review]

**Review by chypriote — APPROVED** [review]

**Pull request merged**